src/server/config/env/runTime.js

```diff
@@ -43,6 +43,24 @@ const runTime = [
     valid: ['development', 'production'],
     defaultValue: 'production',
   },
+  {
+    name: 'ONE_DANGER_DISABLE_CSP',
+    normalize: (input) => {
+      const parsed = input.toLowerCase();
```

Comment:
```suggestion
    normalize: (input) => {
      try {
        return JSON.parse(input.toLowerCase());
      } catch(e) {
        return input;
      }
    },
    validate: (input) => {      
```

Keep the validation separate from the normalization

Using `JSON.parse` like this will allow you to use an actual bool instead of the string in your assertions

Reply:
Hmm, I wonder though. Does it even matter? because you can't save a bool as an env variable? I could be wrong. or is this just to better parse any input

Reply:
Im not too sure about this, in testing it returns a bool  rather than a string. Would this cause problems or does it not matter because it will be saved as a string in env anyways?

Reply:
an issue with checking against a string value `"true"`  is that it increases the risk that a future us will break conditional in a refactor as its common in JS to check that something is truthy rather than true. 

someone may do this refactor and it could slide possibly slide through:
```
if (parsed && process.env.NODE_ENV !== 'development') {
//...
}
```
the issue being any populated string is truthy even `"false"` 
it is a small risk but possible.